function/pkg/operator/triggers_operator.go

```diff
@@ -0,0 +1,142 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-incubator/hydroform/function/pkg/client"
+	"github.com/pkg/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const message = "functionUID"
+
+type triggersOperator struct {
+	items []unstructured.Unstructured
```

Comment:
- ATM we got 2 operators: `GenericOperator` and `TriggersOperator`; the difference between them is the implementation; `TriggersOperator` has additional `Apply` logic that creates extra labels to correlate `Trigger` objects with the `Function` object.

- we decided to use `unstructured.Unstructured` instead types like `FunctionSpec` to avoid introduction of additional dependencies and to avoid problems that spec types versioning may generate